alerts/src/commonTest/kotlin/AlertsTest.kt

```diff
@@ -1,26 +1,55 @@
 package com.splendo.kaluga.test
 
 import com.splendo.kaluga.alerts.Alert
+import com.splendo.kaluga.resources.Color
+import com.splendo.kaluga.resources.colorFrom
+import kotlin.native.concurrent.ThreadLocal
 import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
 import kotlin.test.assertTrue
 
+@ThreadLocal
+object AlertsTestUtils {
+    val whiteColor: Color = colorFrom(255, 255, 255)
```

Comment:
https://github.com/splendo/kaluga/pull/389/files#diff-870e3739b4b4ea8b90cb41e92331f4c1e8efdafa1a0767ec543c34431254e4ceR22

Reply:
Should  I delete `AlertsTestUtils` and use `colorFrom(255, 255, 255)` right where I need it?

Reply:
that too would be better, yes

Reply:
2fb284b